.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,28 @@
+  push:
+    branches: [dev]
+  pull_request:
+    branches: [dev]
+
+jobs:
+  golangci:
+    name: Lint Go code with golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)